.env

```diff
@@ -1,3 +1,5 @@
+GENERATE_SOURCEMAP=false
```

Comment:
This is due to that: 
https://github.com/facebook/create-react-app/discussions/11767